docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1522,14 +1529,21 @@ Default stratifications to all observers should include scenario and input draw.
   * - 12.0
     - * Confirm neonatal all-cause mortality rates match expectation
       * Confirm LBWSG risk effects are working as expected
-    - 
-    - 
+    - * Neonatal all cause mortality rates are within "10% target range," but late neonatal all cause mortality varies more than early neonatal
+      * LBWSG Risk effects and PAF values yet to be directly verified
+    - * `Model 12.0 neonatal notebook <https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/blob/main/verification_and_validation/model_12.0_neonatal_checks.ipynb>`_
   * - 12.1
     - * Confirm neonatal cause-specific mortality risks match expectation for each preterm subcause
       * Confirm LBWSG risk effects are working as expected
       * Check whether neonatal cause-specific mortality risks match expectation for non-preterm causes
     - 
     - 
+  * - 12.1.1
+    - * Confirm neonatal all-cause mortality rates match expectation
```

Comment:
Seems like we should also confirm cause-specific mortality risks!

Reply:
Yes! Thanks - just added